src/testIntegration/java/uk/gov/justice/digital/delius/controller/secure/AttendanceResourceAPITest.java

```diff
@@ -1,55 +1,26 @@
 package uk.gov.justice.digital.delius.controller.secure;
 
```

Comment:
All changes below is moving all integration tests to use the new base class and remove all the duplicated code for

1. Creating tokens
2. Configuring RestAssured
3. JSON writeObject stuff